buf.gen.yaml

```diff
@@ -6,3 +6,7 @@ plugins:
 inputs:
   - proto_file: api/proto/v1/issue.proto
   - proto_file: api/proto/v1/engine.proto
+  - git_repo: https://github.com/ocsf/examples
+    branch: main
+    ref: ba2a49f2bb1faf0c75ece9b6a9c5cd608eafbc67
```

Comment:
They don't use tags but we can still pin a commit so we have some sort of versioning.